no-littering.el

```diff
@@ -314,6 +314,8 @@ directories."
     (eval-after-load 'lookup
       `(make-directory ,(etc "lookup/") t))
     (setq litable-list-file                (var "litable-list.el"))
+    (setq logview-cache-filename           (var "logview-cache.extmap"))
```

Comment:
Why a suffix?

Reply:
I simply put it there because the original value had it. Changed now.